packages/gatsby-theme-store/src/sdk/img/fileManager.ts

```diff
@@ -10,11 +10,17 @@ interface Options {
   quality?: number
 }
 
-export const optimize = (src: string, opts: Options) => {
+export const optimize = (src: string, opts: Options, ratio = 1) => {
```

Comment:
how about:

- renaming `opts` to `imageParams`
- changing `ratio` to an `options` object, to prevent having to add even more arguments if we need to send in different options in the future, and to make it clearer in the function call what "2" means